src/private/MDMPresentationTransitionController.m

```diff
@@ -80,6 +82,23 @@ - (void)setTransition:(id<MDMTransition>)transition {
   return _context;
 }
 
+- (nullable id <UIViewControllerInteractiveTransitioning>)interactionControllerForPresentation:(id <UIViewControllerAnimatedTransitioning>)animator {
+  
+  if ([_transition respondsToSelector:@selector(startWithInteractiveContext:)]) {
+      _interactiveContext = [[MDMViewControllerInteractiveTransitionContext alloc] initWithTransition: _transition];
+    [_transition startWithInteractiveContext:_interactiveContext];
```

Comment:
Ideally we'd put the animated and interactive `start` invocations alongside one another in the transition context instance. Is it possible to move this call there?